skiko/src/awtMain/objectiveC/macos/MetalApi.mm

```diff
@@ -0,0 +1,116 @@
+#ifdef SK_METAL
+
+#import <jawt.h>
+#import <jawt_md.h>
+
+#import <Cocoa/Cocoa.h>
+#import <QuartzCore/QuartzCore.h>
+#import <Metal/Metal.h>
+#import <QuartzCore/CAMetalLayer.h>
+
+#import <GrBackendSurface.h>
+#import <GrDirectContext.h>
+#import <mtl/GrMtlBackendContext.h>
+#import <mtl/GrMtlTypes.h>
+
+#import "MetalDevice.h"
+
+#define MuxGraphicsCard 7
+#define kOpen 0
+#define kGetMuxState 3
```

Comment:
I suspect it's copy-paste. Can we grab all values and/or comments too (probably in separate file)?

Reply:
I don't know) It's already not just magic number, but I have no idea how to verify if it's correct

Reply:
This defines where moved from another file, they are not new ones